client/src/components/_common/SectionHeader/SectionHeader.js

```diff
@@ -31,20 +31,25 @@ import './SectionHeader.module.css';
  * >
  *   Name of Table
  * </SectionHeader>
+ * @example
+ * // a list header (a list can be like a table with no column headers)
+ * <SectionHeader isForList>Name of List</SectionHeader>
  */
 function SectionHeader({
   actions,
   children,
   className,
   isForForm,
-  isForTable
+  isForTable,
+  isForList
```

Comment:
__Explanation__
The `<ManageAccount>` section widgets have "lists" (they are almost all built with tables, but the data is not unequivocally nor intuitively tabular, and it is presented differently enough to merit its own class).